internal/guest/bridge/bridge_v2.go

```diff
@@ -133,18 +136,38 @@ func (b *Bridge) createContainerV2(r *Request) (_ RequestResponse, err error) {
 //
 // This is allowed only for protocol version 4+, schema version 2.1+
 func (b *Bridge) startContainerV2(r *Request) (_ RequestResponse, err error) {
-	_, span := trace.StartSpan(r.Context, "opengcs::bridge::startContainerV2")
+	ctx, span := trace.StartSpan(r.Context, "opengcs::bridge::startContainerV2")
 	defer span.End()
 	defer func() { oc.SetSpanStatus(span, err) }()
-	span.AddAttributes(trace.StringAttribute("cid", r.ContainerID))
+	span.AddAttributes(trace.StringAttribute(logfields.ContainerID, r.ContainerID))
+	entity := log.G(ctx).WithField(logfields.ContainerID, r.ContainerID)
 
 	// This is just a noop, but needs to be handled so that an error isn't
 	// returned to the HCS.
 	var request prot.MessageBase
-	if err := commonutils.UnmarshalJSONWithHresult(r.Message, &request); err != nil {
+	if err = commonutils.UnmarshalJSONWithHresult(r.Message, &request); err != nil {
```

Comment:
I think I changed this function and reverted everything but this change. It shouldn't affect anything since it returns immediately and nothing else uses the pre-return `err`.
I'll change it back.